cmd/describe.go

```diff
@@ -109,25 +110,43 @@ func newDescribeConfig(args []string) describeConfig {
 type description faas.Description
 
 func (d description) Human(w io.Writer) error {
-	fmt.Fprintln(w, d.Name)
+	fmt.Fprintln(w, "Function name:")
+	fmt.Fprintf(w, "  %v\n", d.Name)
+	fmt.Fprintln(w, "Function is built in image:")
+	fmt.Fprintf(w, "  %v\n", d.Image)
+	fmt.Fprintln(w, "Function is deployed as Knative Service:")
+	fmt.Fprintf(w, "  %v\n", d.KService)
+	fmt.Fprintln(w, "Function is deployed in namespace:")
+	fmt.Fprintf(w, "  %v\n", d.Namespace)
 	fmt.Fprintln(w, "Routes:")
+
 	for _, route := range d.Routes {
 		fmt.Fprintf(w, "  %v\n", route)
 	}
-	fmt.Fprintln(w, "Subscriptions (Source, Type, Broker):")
-	for _, s := range d.Subscriptions {
-		fmt.Fprintf(w, "  %v %v %v\n", s.Source, s.Type, s.Broker)
+
+	if len(d.Subscriptions) > 0 {
+		fmt.Fprintln(w, "Subscriptions (Source, Type, Broker):")
+		for _, s := range d.Subscriptions {
+			fmt.Fprintf(w, "  %v %v %v\n", s.Source, s.Type, s.Broker)
+		}
 	}
-	return d.Plain(w)
+	return nil
 }
 
 func (d description) Plain(w io.Writer) error {
 	fmt.Fprintf(w, "NAME %v\n", d.Name)
+	fmt.Fprintf(w, "IMAGE %v\n", d.Image)
+	fmt.Fprintf(w, "KNATIVE SERVICE %v\n", d.KService)
+	fmt.Fprintf(w, "NAMESPACE %v\n", d.Namespace)
+
```

Comment:
Why all caps here (and below)? I know it's what was there before, but why?

Reply:
I think let's be consistent. Just a suggestion.

Reply:
Oh sorry that you had to wait 2 hours for a simple answer... Yes I meant remove all caps to be consistent with other output strings.